CI/package-mudlet-for-windows.sh

```diff
@@ -236,6 +236,7 @@ echo "Copying Lua translation files in..."
 mkdir -p ./translations/lua/translated
 cp -v -p -t ./translations/lua/translated \
     ${GITHUB_WORKSPACE_UNIX_PATH}/translations/lua/translated/mudlet-lua_??_??.json
+cp -v -p -t ./translations/lua ${GITHUB_WORKSPACE_UNIX_PATH}/translations/lua/mudlet-lua.json
```

Comment:
Why don't we have a "en_US" version of the file in the same place as the other "translations"? Is it to make whatever it is that generates the translations easier to iterate through? I recall that I had to do a bit of file juggling to special case the (plurals only) `mudlet_en_US.ts` to `mudlet_en_US.qm` so that it wasn't processed the same as all the other C++ binary translations files...

Reply:
Not sure, we might need it.